include/sound/sof/stream.h

```diff
@@ -49,13 +49,6 @@ enum sof_ipc_frame {
 	/* other formats here */
```

Comment:
typo in commit: "sorce code" -> "source code"

Reply:
updated